protostar/cli/misc.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
```

Comment:
Personally I don't like this module name and in general I am big opponent of making _utility_ modules.

As last resort, this module could be named the same as the main function it exports, like you would do with React components.

Reply:
yeah i thought somebody might provide a better solution than this, so i left it up to reviewers :D